### DISCUSSION TITLE
[theme-styleguide] Add support for object `fontSizes`

### DISCUSSION THREAD

**Discussion by kripod** [LekoArts/gatsby-themes#603] [Ideas / Feature Requests]
## Summary

As [highlighted](https://theme-ui.com/theme-spec#font-sizes-typographic-scale) in the docs of Theme UI, `fontSizes` may be specified as an object:

> Arrays are not required, and plain objects will work as well.

### Motivation

Currently, object syntax is not supported and the resoluting font sizes are too small, especially when demonstrating the contrast between colors:

![image](https://user-images.githubusercontent.com/14854048/73779287-cd163680-478c-11ea-9ee4-57c7519b3099.png)

**Comment by kripod** [comment]
Related: https://github.com/system-ui/theme-ui/pull/646#issuecomment-582034286

**Comment by LekoArts** [comment]
@kripod How does that look like in practice? So how does your `fontSizes` look like?
I don't see anyone using it this way.

**Comment by kripod** [comment]
I’m using sizes like `”sm”`, `”md”` and `”lg”` instead of `1`, `2` and `3`.
